soccer/src/soccer/planning/global_state.hpp

```diff
@@ -0,0 +1,86 @@
+#pragma once
+
+#include <rj_constants/topic_names.hpp>
+#include <rj_msgs/msg/coach_state.hpp>
+#include <rj_msgs/msg/goalie.hpp>
+
+#include "planning/planner/escape_obstacles_path_planner.hpp"
```

Comment:
can you remove this line? (as long as code still compiles) it doesn't seem necessary

Reply:
If I remove any of those includes, the code no longer compiles so I think all the includes are necessary. If you're talking about the blank line, that was added by the style-checker

Reply:
escape obstacles planner definitely isn't needed. compiling probably fails because it's using some include(s) within escape obstacles, but try to track down which ones we actually need. if not, i can try to run a tool I have detects detects necessary includes, in the shop on tuesday, remjnd me.

the reason i care so much is because in the next phase of this EscapeObsatclesPlanner will need to include this file, and we'll have a circular problem.

Reply:
Ok, I'll change the imports to only include necessary files inside escape_obstacles_path_planner

Reply:
I think I've removed all the unnecessary includes now so everything should be good.